pom.xml

```diff
@@ -1000,12 +1000,12 @@
     <module>fractions/wildfly/security</module>
 
     <module>fractions/keycloak</module>
-    <module>fractions/microprofile/microprofile</module>
     <module>fractions/microprofile/microprofile-config</module>
     <module>fractions/microprofile/microprofile-health</module>
     <module>fractions/microprofile/microprofile-jwt</module>
     <module>fractions/microprofile/microprofile-metrics</module>
     <module>fractions/microprofile/microprofile-fault-tolerance</module>
+    <module>fractions/microprofile/microprofile</module>
```

Comment:
What's the reason for this change?

Reply:
Maven builds it in that order anyway, so figured it made sense to arrange them such as /microprofile has the others as dependencies